index.html

```diff
@@ -1 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./src/css/style.css">
+    <title>Apple</title>
+</head>
+<body>
+    <main>
+        <section class="hero-section">
+            
+                <nav>
+            
+                    <div class="nav-items hover border-radius"><a href="#"><i class="fab fa-apple"></i></a></div>
+                    <div class="nav-items hover"><a href="#">Store</a></div>
+                    <div class="nav-items hover"><a href="#">Mac</a></div>
+                    <div class="nav-items hover"><a href="#">iPod</a></div>
+                    <div class="nav-items hover"><a href="#">iPhone</a></div>
+                    <div class="nav-items hover"><a href="#">iPad</a></div>
+                    <div class="nav-items hover"><a href="#">iTunes</a></div>
+                    <div class="nav-items hover"><a href="#">Support</a></div>
+                    <div class="nav-items">
+                        <form >
+                        </form> <input type="text"> <i class="fas fa-search"></i></div>
 
+                </nav>
+            
+            
+            <div class="hero-titles">
+                <div class="title">&nbsp;</div>
+                <div class="sub-titles">&nbsp;</div>
+                <div class="last-titles">&nbsp;</div>
+
+            </div>
+        </section>
+        <section class="product-section">
+
+            <div class="product-one">&nbsp;</div>
+            <div class="product-two">&nbsp;</div>
+            <div class="product-three">&nbsp;</div>
+            <div class="product-four">&nbsp;</div>
```

Comment:
- [ ] It contains four elements which are links (to other articles or #). The four elements at the bottom of the page(i.e the products) should be links(`<a href="#">`)